build/gulpfile.hygiene.js

```diff
@@ -104,7 +104,8 @@ const indentationFilter = [
 	'!extensions/integration-tests/testData/**',
 	'!extensions/big-data-cluster/src/bigDataCluster/controller/apiGenerated.ts',
 	'!extensions/big-data-cluster/src/bigDataCluster/controller/clusterApiGenerated2.ts',
-	'!resources/linux/snap/electron-launch'
+	'!resources/linux/snap/electron-launch',
+	'!build/actions/**/dist/*.js'
```

Comment:
If you want to remove hygiene for everything. Should you add it to the `all` rather than adding it to every filter? (these are waterfall filters).

Reply:
I really need to look at the code closer :D 

Thanks!